descriptors/ModuleDescriptor-template.json

```diff
@@ -196,6 +196,11 @@
           "methods": ["PUT"],
           "pathPattern": "/inn-reach/central-servers/{centralServerId}/patron-type-mappings",
           "permissionsRequired": ["inn-reach.patron-type-mappings.collection.put"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inn-reach/central-servers/{centralServerId}/contributions/start-initial-contribution",
```

Comment:
Let's align this URL with contribution retrieval. I suggest to use:
`POST /inn-reach/central-servers/{centralServerId}/contributions/ `

w/o any additional URL parts, because POST already means that a new resource is created (in our case new contribution is being started)